src/asm/main.c

```diff
@@ -36,6 +36,10 @@
 #include "helpers.h"
 #include "version.h"
 
+// There are known, non-trivial to fix leaks. We would still like to have `make develop'
+// detect memory corruption, though.
+const char* __asan_default_options(void) { return "detect_leaks=0"; }
```

Comment:
I'd rather have those be defined in the test files (`export ASAN_OPTIONS="detect_leaks=0"` IIRC)